src/message_bus.zig

```diff
@@ -622,12 +627,12 @@ fn MessageBusImpl(comptime process_type: ProcessType) type {
                         //
                         // TODO: Investigate differences between shutdown() on Linux vs Darwin.
                         // Especially how this interacts with our assumptions around pending I/O.
-                        const rc = os.system.shutdown(connection.fd, os.SHUT_RDWR);
+                        const rc = os.system.shutdown(connection.fd, os.SHUT.RDWR);
                         switch (os.errno(rc)) {
-                            0 => {},
-                            os.EBADF => unreachable,
-                            os.EINVAL => unreachable,
-                            os.ENOTCONN => {
+                            os.E.SUCCESS => {},
+                            os.E.BADF => unreachable,
+                            os.E.INVAL => unreachable,
+                            os.E.NOTCONN => {
```

Comment:
We could use enum literals for errno handling here and elsewhere if you like.
```suggestion
                            .SUCCESS => {},
                            .BADF => unreachable,
                            .INVAL => unreachable,
                            .NOTCONN => {
```